bin/replicate-mongodb.sh

```diff
@@ -16,22 +16,22 @@ bucket="govuk-integration-database-backups"
 mongo_version=3.6
 
 case "$app" in
-  "authenticating-proxy")
-    hostname=router_backend
-    database=authenticating_proxy_production
-    ;;
   "router"|"draft-router")
-    hostname=router_backend
+    hostname=mongo-router
     database="${app//-/_}"
     wait_for_rs=1
     ;;
-  "asset-manager")
-    hostname=mongo
-    database=govuk_assets_production
+  "licence-finder")
+    hostname=mongo-normal
+    database=licence_finder_production
     ;;
-  "manuals-publisher"|"publisher"|"specialist-publisher")
-    hostname=mongo
-    database=govuk_content_production
+  "licensify")
+    hostname=mongo-licensing
+    database=licensify
+    ;;
+  "content-store"|"draft-content-store")
+    hostname=mongo-api
+    database=content_store_production
```

Comment:
This should dynamically choose the draft or live backup (like we do for Router above):

```suggestion
    database="${app//-/_}_production"
```

(Also matches line 38)